tests/Node.py

```diff
@@ -1389,6 +1399,19 @@ def isLibAdvancing():
             return self.getIrreversibleBlockNum() > currentLib
         return Utils.waitForBool(isLibAdvancing, timeout)
 
+    def waitForProducer(self, producer, timeout=None, exitOnError=False):
+        if timeout is None:
+            # default to the typical configuration of 21 producers, each producing 12 blocks in a row
+            timeout = 21 * 12;
```

Comment:
Each producing 12 blocks in a row at half second intervals.  21 * 6.  `Utils.waitForBool()` expects seconds.